index.html

```diff
@@ -207,64 +208,91 @@ <h2>Git Workflow</h2>
                 <h2>A bit about git internals</h2>
                 <p>
-                <ul>
-                    <li class="fragment">Commits are actually SHA1 hashes that creates the distinction</li>
-                    <li class="fragment">All commits in repo are nodes of a tree</li>
-                    <li class="fragment">.git folder contains:
-                        <ul>
-                            <li>config – Configuration files</li>
-                            <li>logs/* - All the logs for commits</li>
-                            <li>objects/* - The object repository</li>
-                            <li>refs/remotes/* - tracking other remotes</li>
-                            <li>Index – index cache (staging area)</li>
-                            <li>HEAD – points to the latest commit on current branch</li>
-                        </ul>
-                    </li>
-                </ul>
+                    <ul>
+                        <li class="fragment">Commits are actually SHA1 hashes that creates the distinction</li>
+                        <li class="fragment">All commits in repo are nodes of a tree</li>
+                        <li class="fragment">.git folder contains:
+                            <ul>
+                                <li>config – Configuration files</li>
+                                <li>logs/* - All the logs for commits</li>
+                                <li>objects/* - The object repository</li>
+                                <li>refs/remotes/* - tracking other remotes</li>
+                                <li>Index – index cache (staging area)</li>
+                                <li>HEAD – points to the latest commit on current branch</li>
+                            </ul>
+                        </li>
+                    </ul>
                 </p>
             </section>
             <section>
                 <h2>Where to go from here?</h2>
                 <p>
-                <ul>
-                    <li><a href="http://www.git-scm.com">www.git-scm.com</a></li>
-                    <li>Learn git-flow</li>
-                </ul>
+                    <ul>
+                        <li>Implement git and GitHub in your future projects.</li>
+                        <li>Contribute to major upstream open-source projects like Mozilla, Django, Linux kernel and many more.</li>
+                        <li>Sharpen your skills and learn new technologies by solving issues.</li>
+                        <li>Widen your network and find fun new side-projects.</li>
+                        <li>Use your GitHub profile to showcase projects on your resume.</li>
+                    </ul>
+                </p>
+            </section>
+            <section>
+                <h2>Links for learning more</h2>
+                <p>
+                    <ul>
+                        <li>Documentation for reference - <a href="http://www.git-scm.com/">http://www.git-scm.com/</a> </li>
```

Comment:
Including too many links for beginners would make them go absolutely no where. You can mention them as Resources, but keep one slide as it is, which was, "where to go from here" --- And say git-scm.

Reply:
Liked the idea. Implemented,